sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/TelegramChannelProperties.cs

```diff
@@ -30,14 +29,14 @@ public TelegramChannelProperties()
         /// <summary>
         /// Initializes a new instance of the TelegramChannelProperties class.
         /// </summary>
+        /// <param name="isEnabled">Whether this channel is enabled for the
+        /// bot</param>
         /// <param name="accessToken">The Telegram access token. Value only
         /// returned through POST to the action Channel List API, otherwise
         /// empty.</param>
-        /// <param name="isEnabled">Whether this channel is enabled for the
-        /// bot</param>
         /// <param name="isValidated">Whether this channel is validated for the
         /// bot</param>
-        public TelegramChannelProperties(string accessToken, bool isEnabled, bool? isValidated = default(bool?))
+        public TelegramChannelProperties(bool isEnabled, string accessToken = default(string), bool? isValidated = default(bool?))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
will add fallback constructor for back compat.